store/tikv/retry/config.go

```diff
@@ -89,6 +89,7 @@ var (
 	BoPDRPC      = NewConfig("pdRPC", &metrics.BackoffHistogramPD, NewBackoffFnCfg(500, 3000, EqualJitter), tikverr.NewErrPDServerTimeout(""))
 	// change base time to 2ms, because it may recover soon.
 	BoRegionMiss        = NewConfig("regionMiss", &metrics.BackoffHistogramRegionMiss, NewBackoffFnCfg(2, 500, NoJitter), tikverr.ErrRegionUnavailable)
+	BoRegionScheduling  = NewConfig("regionScheduling", &metrics.BackoffHistogramRegionScheduling, NewBackoffFnCfg(2, 500, NoJitter), tikverr.ErrRegionUnavailable)
```

Comment:
> Add the `regionScheduling` backoff configuration for backoff due to region scheduling which is separated from the `regionMiss`. The aim is to reduce backoff sleep time.

`BoRegionScheduling` config is the same as `BoRegionMiss`?

Reply:
Yes. It reduces the backoff sleep time by dividing the backoff times to different configs.

Reply:
Ah, I see...